src/util/storybook-docs/Table.tsx

```diff
@@ -58,17 +68,22 @@ const TableHeadLayout = ({ headers = [] }) => (
   </thead>
 );
 
-TableHeadLayout.propTypes = {
-  headers: PropTypes.array,
-};
+export interface TableLayoutProps {
+  rows?: Array<Array<React.ReactNode | object>>;
```

Comment:
Yikes 😁 yeah there is definitely some complexity with this component

Reply:
yep, later on i'd like to have an initiative of reducing complexity on components that allow us to do so.